### PR TITLE
koji_promote: encode logs to UTF-8

### PR DESCRIPTION
**Pull request by vrutkovs**
Make sure logs are converted to UTF-8 correctly

### CODE REVIEW

File: tests/plugins/test_koji_promote.py [hunk 1]
```diff
@@ -223,15 +223,15 @@ def mock_environment(tmpdir, session=None, name=None,
     (flexmock(OSBS)
         .should_receive('get_build_logs')
         .with_args(BUILD_ID)
-        .and_return('build logs'))
+        .and_return(u'build logs - \u2018 \u2017 \u2019'))
```
Comment (lcarva) [inline tests/plugins/test_koji_promote.py]: do you need the `u''` prefix here? This file uses `unicode_literals`.

Comment (vrutkovs) [inline tests/plugins/test_koji_promote.py]: Right, I don't really need it

File: atomic_reactor/plugins/exit_koji_promote.py [hunk 2]
```diff
@@ -339,7 +339,8 @@ def get_logs(self):
             # Deleted once closed
             logfile = NamedTemporaryFile(prefix=self.build_id,
                                          suffix=".log",
-                                         mode='w')
+                                         mode='w',
+                                         encoding='UTF-8')
```
Comment (lcarva) [inline atomic_reactor/plugins/exit_koji_promote.py]: Only available in python3. In python 2, there's codec.open, but not sure about temp file.

**Comment by vrutkovs** [comment]
Tested with rsyslog and previously failing cfme build - worked on both.

Not sure why coverage doesn't like this change though

**Comment by lcarva** [comment]
I think the coverage drop is because it's using the 3.X build result, not the 2.7 build result. Maybe it just picks a random build? Anyways, I rebuilt it and now coveralls is happy.

**Pull request merged**